config/F32.config

```diff
@@ -58,21 +58,21 @@ process {
     }
     withName: call_sSNV_Mutect2 {
         cpus = 1
-        memory = 3.GB
+        memory = 2.GB
         retry_strategy {
             memory {
                 strategy = 'add'
-                operand = 3.GB
+                operand = 2.GB
```

Comment:
Is the reduction here for Mutect2 intended?

Reply:
Yes, looking at all the recent trace files I think 3.GB was overly cautious.    Generally only one of the 50 scatters is susceptible to a memory failure and 2.GB is still usually more than enough.  Plus the process is relatively short so the rerun is easily absorbed.